content/en/docs/how-tos/cluster-claim.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Creating a Cluster Pool"
+description: How to create a cluster pool from which a job can claim a cluster.
+---
+
+DPTP integrates Hive into the OpenShift test platform to manage the cluster pools and clusters in them. There is a defined pool [ci-cluster-pool](https://github.com/openshift/release/tree/master/clusters/app.ci/ci-cluster-pool) from which can be used to claim an OpenShift cluster for a test. In this page, we document how to create a new cluster pool with the cloud account provided by a CI user.
+
+## Prerequisites
+
+*  Choose a [Hive](https://github.com/openshift/hive#supported-cloud-providers) supported cloud for the pool.
+* [Configure the cloud account](https://docs.openshift.com/container-platform/4.7/installing/installing-preparing.html).
+
+## Create a cluster pool
+
+Create a folder in `release/clusters/app.ci/` with the manifests for the pool. See [the examples](https://github.com/openshift/release/tree/master/clusters/app.ci/ci-cluster-pool) for the existing `ci-cluster-pool`.
```

Comment:
Hrm, maybe we want to group them? IMO, we should optimize for a case where every pool-owning group has multiple pools, and we have multiple (or even many) groups that manage their pools. So IMO we want this:

```
clusters/
  app.ci/
    pools/
      groupA/
        pool1.yaml
        pool2.yaml
        ...
      groupB/
        pool1.yaml
        pool2.yaml
```

and not 

```
clusters/
  app.ci/
    groupA/
      pool1.yaml
      pool2.yaml
      ...
    groupB/
      pool1.yaml
      pool2.yaml
```

Reply:
Also `release/` is not a path in `openshift/release`.

Reply:
https://github.com/openshift/release/pull/17950